### DISCUSSION TITLE
Allow users to customize order of sprint poker activities

### DISCUSSION THREAD

**Discussion by drewhousman1010** [ParabolInc/parabol#6128] [Feedback]
A [user](https://app.hubspot.com/contacts/3888472/contact/64981601) 🔒 writes:

> For the sprint poker, I like to prepare in advance which issues to bring in from jira, before the meeting starts.  So I start the meeting, and jump to the issue selection part, and then go back to the icebreaker and invite people. Maybe some way to plan a meeting with issue selection in the planning could be good =)

The sense from the devs was that this user wants a meeting order to be different, first scoping phase, then icebreakers, and then an estimation phase.

In addition we think they want a meeting with an initial planning state (like a drafting state) where they can add the issues before they officially open the meeting and people are invited.

**Comment by fiddur** [comment]
Yes, letting the facilitator select issues to refine/estimate before inviting participants would be great.

To clarify why I want this: The issues displayed from jira are not always in the order of the backlog from which issues will be taken into coming sprint.  This is especially true when it comes to subtasks I think, since only issues are ordered in backlog but it is the subtasks that are estimated.  So, when preparing the poker planning, I jump between jira and parabol, searching by title mostly, and that work is a waste of time for participants to observe.

**Reply by tianrunhe** [reply]
Hi @fiddur, thanks for the suggestion! I'd like to offer a intermediate solution, not perfect but let me know if it suits your need at this moment! 
Did you know you can always go back to Ice Breaker phase even you have proceeded to Scoping or Estimation phases? Here's a demo GIF: you can  see that I'm already in Estimation phase with two issues added, but I can still go back to Ice Breaker. Although it appears that the phase has already ended with streaklines crossed members' names, but as the facilitator I can still go through this phase one by one.

![2022-02-25 10 35 43](https://user-images.githubusercontent.com/1879975/155769923-a34839a7-38b4-4fa7-bbf0-1d1b9de2beae.gif)

So my suggested sequence of actions to cater for your use case are:
1. Start the poker meeting well before your scheduled team planning time (remember to select include Ice Breaker)
2. You don't need to actively invite team members at this time. They might get notifications from Parabol and join but that's OK. Let them know you just start this meeting in advance for preparation
3. Skip the Ice Breaker phase and proceed to Add as many issues as you'd like to. Take enough time you need. No pressure 😂.
4. When you finish adding all the issues and it's time for your team's sync planning time, ask team members to join the meeting
5. Go back to the Ice Breaker phase as shown in the GIF above and your team can share their answers! 🎉
